docs/integrate/airflow/index.md

```
@@ -38,40 +38,45 @@ to fit the level of abstraction that suits your environment.
 [![Astronomer logo](https://logowik.com/content/uploads/images/astronomer2824.jpg){w=180px}](https://www.astronomer.io/)
 ```
 
-[Astro][Astronomer] is the best managed service in the market for teams on any step of their data
-journey. Spend time where it counts.
+[Astro][Astronomer] is a managed Airflow service.
```

markdownlint-cli2 (0.17.2)

Reference links and images should use a label that is defined
Missing link or image reference definition: "astronomer"

(MD052, reference-links-images)